src/tests/unit/tests/background/tab-context-broadcaster.test.ts

```diff
@@ -1,25 +1,68 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { Mock, MockBehavior, Times } from 'typemoq';
+import { It, Mock, MockBehavior, Times } from 'typemoq';
 
 import { TabContextBroadcaster } from 'background/tab-context-broadcaster';
+import { Logger } from 'common/logging/logger';
 import { BrowserAdapter } from '../../../../common/browser-adapters/browser-adapter';
 import { StoreUpdateMessage } from '../../../../common/types/store-update-message';
 
-describe('TabContextBroadcasterTest', () => {
-    test('getBroadcastMessageDelegate', () => {
-        const testTabId = 1;
-        const testMessage = { someData: 1 } as any;
-        const expectedMessage = { tabId: testTabId, ...testMessage } as StoreUpdateMessage<any>;
+describe('TabContextBroadcaster', () => {
+    describe('getBroadcastMessageDelegate', () => {
+        it('produces the expected BrowserAdapter calls when invoked', async () => {
+            const testTabId = 1;
+            const testMessage = { someData: 1 } as any;
```

Comment:
can `someData` be a different number, not `1` so no reader confuse this number to be `testTabId`?